pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[tool.poetry]
+name = "moviecenter"
+version = "0.1.0"
+description = ""
+authors = ["Eli"]
+
+[tool.poetry.dependencies]
+python = "3.10"
```

Comment:
Not relevant because that's an app, but if it was a package it would be better to specify `^3.10` instead

Reply:
pytest requires this explicitly (to compare against the required version of the dependencies)